server/config/routes.js

```diff
@@ -84,10 +84,39 @@ const financialPatient = require('../controllers/finance/patient');
 const dashboardDebtors = require('../controllers/dashboard/debtorGroups');
 const stats = require('../controllers/dashboard/stats');
 
+
+//email report routes
+const report_group=require('../controllers/report-group');
+const email_report=require('../controllers/email-report');
+const weeklySummaryReport=require('../controllers/email-report/weeklySummaryReport');
+const mail=require('../controllers/report-group/mail');
+const scheduler=require('../controllers/report-group/scheduler');
 // expose routes to the server.
 exports.configure = function configure(app) {
   winston.debug('Configuring routes');
 
+
+//this is the scheduler config for send reports emailing 
+scheduler.setScheduler();
```

Comment:
This should be in the `server/config/express.js` file.  The `server/config/routes.js` file should only contain routes.